style.css

```diff
@@ -1,20 +1,117 @@
-*{
-    margin: 0;
-    padding: 0;
-    box-sizing: border-box;
-    height: 100vh;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
 }
-.box{
-    background-color: blueviolet;
-    border: 1px solid black;
-    width: 200px;
-    height: 50vh;
-    flex: 1;
+
+@media only screen and (max-width: 768px) {
+  body {
+    height: 100vh;
+    width: 100%;
+    left: 0;
+    top: 0;
+    background-color: #fff;
+  }
+
+  nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 4vh 5vh;
+  }
+
+  .top-logo h2 {
+    font-family: 'Poppins', sans-serif;
+    font-size: 22px;
+    color: #6070ff;
+  }
```

Comment:
- [ ] Please import `Poppins` from [Google fonts](https://fonts.google.com/). While you are using `'Poppins'` font-family, browser is going with default `sans-serif` because `Poppins` has not been imported in CSS. Please make sure your fonts match with [Figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=rTgMr8v8dtgLyJg0-0)

**_Your design_**

<img width="268" alt="Screenshot 2023-02-16 at 11 24 27 AM" src="https://user-images.githubusercontent.com/44798044/219295878-9c774f6f-1dbd-4d87-bd5e-de34da78f75b.png">

**_Figma Design_**

<img width="349" alt="Screenshot 2023-02-16 at 11 18 47 AM" src="https://user-images.githubusercontent.com/44798044/219296205-a7c6d4e5-153f-42ab-9f9c-47c0554b67ae.png">